src/System.Private.CoreLib/src/System/Threading/Thread.CoreCLR.cs

```diff
@@ -111,7 +112,7 @@ public sealed partial class Thread
         ** ThreadBaseObject to maintain alignment between the two classes.
         ** DON'T CHANGE THESE UNLESS YOU MODIFY ThreadBaseObject in vm\object.h
         =========================================================================*/
-        private ExecutionContext m_ExecutionContext;    // this call context follows the logical thread
+        internal ExecutionContext m_ExecutionContext;    // this call context follows the logical thread
```

Comment:
Change this to ‘_’ prefix to make it follow coding conventions? It is ‘_’ in CoreRT already.

Reply:
Added a commit to clean up style in the file.